modules/phase_field/src/materials/ComputePolycrystalElasticityTensor.C

```diff
@@ -48,9 +48,6 @@ ComputePolycrystalElasticityTensor::computeQpElasticityTensor()
   const std::vector<std::pair<unsigned int, unsigned int> > & active_ops = _grain_tracker.getElementalValues(_current_elem->id());
```

Comment:
Sorry I know I'm sidetracking your PR. This is actually a separate issue and doesn't need to be taken care of here but the discussion is still important.